engine/src/main/java/org/entando/entando/aps/system/services/entity/model/EntityDto.java

```diff
@@ -73,7 +73,7 @@ public void fillEntity(IApsEntity prototype, ICategoryManager categoryManager, B
         prototype.setDescription(this.getDescription());
         prototype.setMainGroup(this.getMainGroup());
         if (null != this.getGroups()) {
-            this.getGroups().stream().forEach(i -> prototype.addGroup(typeCode));
+            this.getGroups().stream().forEach(groupCode -> prototype.addGroup(groupCode));
```

Comment:
Does this need to be streamed? prototype.getGroups().addAll(this.getGroups());

Or expose an addList method.

Reply:
Done